src/core/libmaven/PeakGroup.cpp

```diff
@@ -254,13 +238,20 @@ void PeakGroup::setCompound(Compound* compound)
 void PeakGroup::addPeak(const Peak &peak)
 {
 	peaks.push_back(peak);
-	peaks.back().groupNum = groupId;
+    peaks.back().groupNum = _groupId;
 }
 
 void PeakGroup::setSlice(const mzSlice& slice)
 {
     _slice = slice;
     _sliceSet = true;
+    if (!_slice.isotope.isNone()) {
+        tagIsotope(_slice.isotope.name,
+                   _slice.isotope.mass,
+                   _slice.isotope.abundance);
+        isotopeC13count = _slice.isotope.C13;
```

Comment:
I doubt `isotopeC13count` is used anywhere. You can probably remove it.

Reply:
Right. Removing.